Friend/firmware/firmware_v1.0/prj_xiao_ble_sense_devkitv1.conf

```diff
@@ -90,6 +90,11 @@ CONFIG_LOG_DEFAULT_LEVEL=3
 # CONFIG_BT_RX_STACK_SIZE=4096
 # CONFIG_BT_CTLR_RX_PRIO_STACK_SIZE=4096
 
+CONFIG_LSM6DSL=y
+CONFIG_LSM6DSL_ENABLE_TEMP=n
```

Comment:
How comes some of these are not on the devkit2 proj file? If not needed remove or add to both files

```
CONFIG_LSM6DSL_ENABLE_TEMP=n
CONFIG_LSM6DSL_TRIGGER_GLOBAL_THREAD=y
CONFIG_SENSOR=y
```